docs/DeveloperGuide.md

```diff
@@ -6,33 +6,360 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### User Interface (UI) - Class Implementation
+![img.png](img.png) <br>
```

Comment:
![image](https://user-images.githubusercontent.com/88373092/228757820-52b768c6-a066-4959-99c2-83db94d389ed.png)

Perhaps that it would be better to use the standard notation for the UML diagram. To do that you could use the notation `hide circle` and `skinparam classAttributeIconSize 0` in your .puml file.